micro_sam/models/sam_3d_wrapper.py

```diff
@@ -1,15 +1,15 @@
-from typing import Type
+from typing import Any, List, Dict, Type
 
 import torch
 import torch.nn as nn
 
 from segment_anything.modeling.image_encoder import window_partition, window_unpartition
 from segment_anything.modeling import Sam
 
-from .util import get_sam_model
+from ..util import get_sam_model
 
 
-def get_3d_sam_model(
+def get_sam_3d_model(
```

Comment:
I changed these names to match the order of `sam_3d` we use everywhere else, cc @lufre1